src/EditorFeatures/Core/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -119,6 +426,14 @@ public void ExecuteCommand(TypeCharCommandArgs args, Action nextHandler, Command
         if (args.SubjectBuffer.IsInLspEditorContext())
             return;
 
+        ThreadingContext?.JoinableTaskFactory.Run(async () =>
+            {
+                if (_suggestionServiceBase is not null)
```

Comment:
this is `IAsyncDisposable`, we probably need to dispose it when the textView is closed